types/node-telegram-bot-api/index.d.ts

```diff
@@ -161,7 +161,6 @@ declare namespace TelegramBot {
         reply_to_message_id?: number | undefined;
         reply_markup?: InlineKeyboardMarkup | ReplyKeyboardMarkup | ReplyKeyboardRemove | ForceReply | undefined;
         protect_content?: boolean | undefined;
-        allow_sending_without_reply?: boolean | undefined;
```

Comment:
Is this line intentional? Deletion of this line failed the tests (See GitHub pipeline).

Reply:
Is not intentional, will restore it in a moment. Thanks!